src/sidebar/util/service-context.js

```diff
@@ -80,7 +80,7 @@ export function withServices(Component) {
       }
 
       if (!(service in props)) {
-        services[service] = $injector.get(service);
+        services[service] = injector.get(service);
```

Comment:
Will these always be services (only, versus other things that could ostensibly be registered)?

Reply:
Currently we have services and "environment" objects (settings from the host page or h instance, whether or not the app is the sidebar etc.).  We could potentially use it in future for runtime implementation switching of certain dependencies depending on the application (client, activity pages etc.).